### ISSUE TITLE
Request examples for data organization

### ISSUE THREAD

**Issue by Cli98** [rwightman/efficientdet-pytorch#22]
Hi @rwightman 

Thank you for your high quality work! Here I want to ask if you can provide examples for dataset organizations to run your pipeline.

For example, in coco detection, we have train_image/val_image and train_annotation.json/val_annotation.json (just ignore test set shortly), then how should I place them to make sure it is acceptable for your program to run?

Thank you for your help.

**Comment by rwightman** [comment]
@Cli98 it expects a fairly standard mscoco layout for 2017 datasets, I will make it a bit more configurable at some point for custom (coco like) datasets, but should be hackable fairly quickly as is. If you extract as per the README I'm pretty sure it ends up in the right layout, but in case you've had it extracted already....

The root coco folder that you pass in as arg should contain these folders:
```
annotations
train2017
val2017
test2017 (if you want to run test/test-dev)
```

The {train, test, val} folders are flat folders containing all trani/test/val images for 2017 dataset. The `annotations` subfolder should contain all the json files, currently relevant ones include:
```
instances_train2017.json
instances_val2017.json
image_info_test-dev2017.json (if you want to run test-dev)
image_info_test2017.json (if you want to run test)
```

**Comment by Cli98** [comment]
> @Cli98 it expects a fairly standard mscoco layout for 2017 datasets, I will make it a bit more configurable at some point for custom (coco like) datasets, but should be hackable fairly quickly as is. If you extract as per the README I'm pretty sure it ends up in the right layout, but in case you've had it extracted already....
> 
> The root coco folder that you pass in as arg should contain these folders:
> 
> ```
> annotations
> train2017
> val2017
> test2017 (if you want to run test/test-dev)
> ```
> 
> The {train, test, val} folders are flat folders containing all trani/test/val images for 2017 dataset. The `annotations` subfolder should contain all the json files, currently relevant ones include:
> 
> ```
> instances_train2017.json
> instances_val2017.json
> image_info_test-dev2017.json (if you want to run test-dev)
> image_info_test2017.json (if you want to run test)
> ```

Excellent. That's exactly what I expected. Appreciate your help man.
